scripts/run-ui-tests.sh

```diff
@@ -132,22 +130,24 @@ ANDROID_RETURN_CODE=$(initialize_tests $ANDROID_APP $ANDROID_DEVICES $TEST_APK $
 ANDROID_TEST_RUN_ID=$(get_test_run_id $ANDROID_INFORMATION_FILE)
 echo $ANDROID_TEST_RUN_ID
 print_initialization_results $ANDROID_RETURN_CODE "$ANDROID_PLATFORM_NAME" "$ANDROID_PORTAL_URL" "$ANDROID_TEST_RUN_ID"
+cat $ANDROID_INFORMATION_FILE
 rm $ANDROID_INFORMATION_FILE
 
 # Run iOS tests
 echo "Initializing iOS tests..."
 IOS_RETURN_CODE=$(initialize_tests $IOS_APP $IOS_DEVICES $TEST_IPA $IOS_INFORMATION_FILE)
 IOS_TEST_RUN_ID=$(get_test_run_id $IOS_INFORMATION_FILE)
 print_initialization_results $IOS_RETURN_CODE "$IOS_PLATFORM_NAME" "$IOS_PORTAL_URL" "$IOS_TEST_RUN_ID"
+cat $IOS_INFORMATION_FILE
 rm $IOS_INFORMATION_FILE
 
 # If iOS or Android tests failed to be initiated, exit failure. Otherwise exit success
 if [ $IOS_RETURN_CODE -ne 0 ] || [ $ANDROID_RETURN_CODE -ne 0 ]; then	
     exit 1
 fi
 
-# If script is running in bitrise environment, upload test run IDs to Azure Storage
-if ! [ -z ${IN_BITRISE+x} ]; then # Then we are in bitrise environment
+# If enabled, upload test run IDs to Azure Storage
+if [ "$UPLOAD_RUN_IDS_TO_STORAGE" == "true" ]; then # Then we are in bitrise environment
```

Comment:
i guess the comment about being in bitrise can be removed

Reply:
oops